operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -16,6 +16,14 @@ type MultiClusterObservabilitySpec struct {
 	// Advanced configurations for observability
 	// +optional
 	AdvancedConfig *AdvancedConfig `json:"advanced,omitempty"`
+	// Size read path of your Observability instance
+	// +optional
+	ReadTShirtSize TShirtSize `json:"readTShirtSize,omitempty"`
+	// Size write path of your Observability instance
+	// Some components are shared with the read path. In case of common components,
+	// we will use write path size.
+	// +optional
+	WriteTShirtSize TShirtSize `json:"writeTShirtSize,omitempty"`
```

Comment:
I wonder are we adding complexity here by differentiating between the read and write path? It stands to reason that if you want to ingest metrics at a certain scale then you would also need to have the infrastructure to query those metrics!

I am just considering that from the offset we may be making this more difficult for ourselves than it needs to be, wdyt?

Reply:
So initially I was thinking of doing just a single size. But the nature of monitoring, is that you'd ingest quite a lot of metrics, but then query only a fraction most of the time whilst debugging/etc. You might have some heavy rules/dashboards, but that'd be the max. 

So I guess, instead of sizing both ingest and query in similar scale, a user could potentially save resources by scaling down query?

Reply:
Ack, makes sense then! Will unify to one value for now, and we can modify later.